### ISSUE TITLE
Error when running extension

### ISSUE THREAD

**Issue by MayuDev** [ivangabriele/vscode-heroku#220]
Something went wrong while trying to list your currents Heroku apps.

**Comment by pelted** [comment]
I had the same issue, but upgraded my cli to `heroku/7.39.5` and that part resolved.

**Comment by MayuDev** [comment]
> I had the same issue, but upgraded my cli to `heroku/7.39.5` and that part resolved.

That definitely helped but now I'm getting these 2 messages `Your current workspace is now linked to the "jonin-2" Heroku app.` and `Something went wrong while linking your Heroku app: "jonin-2".`

**Comment by pelted** [comment]
Yeah I don't get that far. My apps are in a teams account and although the extension uses the -A flag to list apps, nothing shows up. Seems like there are larger issues here.

**Comment by MayuDev** [comment]
Hopefully, this can be fixed this

**Comment by justgage** [comment]
Yeah, I think it can't parse the list when the upgrade message shows up. Updating fixed it for me.
